idb/postgres/postgres_migrations.go

```diff
@@ -22,13 +23,33 @@ import (
 
 const migrationMetastateKey = "migration"
 
+func init() {
+	migrations = []migrationStruct{
+		{m0fixupTxid, false, "Recompute the txid with corrected algorithm."},
+		{m1fixupBlockTime, true, "Adjust block time to UTC timezone."},
+		{m2apps, true, "Update DB Schema for Algorand application support."},
+		{m3acfgFix, false, "Recompute asset configurations with corrected merge function."},
+		{m4cumulativeRewardsDBUpdate, true, "Update DB Schema for cumulative account reward support."},
+		{m5accountCumulativeRewardsUpdate, true, "Compute cumulative account rewards for all accounts."},
+	}
+}
+
+
 type MigrationState struct {
 	NextMigration int `json:"next"`
 
-	// Used for a long migration to checkpoint progress
+	// NextRound used for m0 to checkpoint progress.
 	NextRound int64 `json:"round,omitempty"`
 
+	// NextAssetId used for m3 to checkpoint progress.
 	NextAssetId int64 `json:"assetid,omitempty"`
+
+	// NextAccount used for m5 to checkpoint progress.
+	NextAccount []byte `json:"nextaccount,omitempty"`
+
+	// Note: a generic "data" field here could be a good way to deal with this growing over time.
```

Comment:
I think MigrationState as json with fields only present if used satisfies this?
Also I think it wouldn't be bad for migrations to re-use NextFoo if they wind up iterating over that field similar to how a prior migration did.

Reply:
Currently there's no cleanup mechanism, so they keep adding up. That also means we can't reuse a field because there isn't a good way of telling whether the individual migration is just starting, or resuming from where the earlier migration left off.

Reply:
Looks like I might have been wrong about that comment. Added the cleanup code to my migration so this will continue to be an option in the future.